packages/zoe/src/contractSupport/auctions.js

```diff
@@ -1,64 +1,51 @@
-export const secondPriceLogic = (bidAmountMath, bidOfferHandles, bids) => {
+export const secondPriceLogic = (bidAmountMath, bidSeats) => {
   let highestBid = bidAmountMath.getEmpty();
   let secondHighestBid = bidAmountMath.getEmpty();
-  let highestBidOfferHandle;
-  // eslint-disable-next-line array-callback-return
-  bidOfferHandles.map((offerHandle, i) => {
-    const bid = bids[i];
+  let highestBidSeat;
+
```

Comment:
Since the loop is for computing the value of `highestBidSeat`, this space shouldn't be here.